docs/scenario_files.md

```diff
@@ -0,0 +1,117 @@
+# Scenario files
+Using a scenario file to configure tests offers several advantages over setting the parameters via the command line. Scenario files provide more flexibility in defining tests, greater ease of modifying tests, improved test reproducibility, and enable the creation of more complex test runs. Once a scenario file is created, that scenario can also easily be exported for future use.
+
+## Scenario file format
+
+    global:
+        <global options>
+    systems:
+        system1:
+            <system1 options>
+
+The outline above can also easily be extended to incorporate multiple systems within a single scenario, like so:
+
+    global:
+        <global options>
+    systems:
+        system1:
+            <system1 options>
+        system2:
+            <system2 options>
+
+## Scenario file options
+The options in the parameters file maps directly back to the CLI options, minus the dash.
+
+
+## Using scenario files
+
+Scenario files are placed within your Zathras working directory, and can be given any name. For example, given a scenario file `my_scenario`, the following command would begin a Zathras run using that scenario file:
+
+    ./bin/burden --scenario my_scenario
+
+## Examples
+
+### Scenario file for bare metal
+#### Example 1 
+This scenario would run the STREAM benchmark on a local bare metal system named "test_sys".
```

Comment:
Probably best to mention the need for a `local_config` here, with a link to relevant documentation if available.